docs/rancher/rke1-cluster.md

```diff
@@ -51,3 +51,26 @@ Users can create an RKE1 Kubernetes cluster from the **Cluster Management** page
 1. Click **Create**.
 
 ![create-rke-harvester-cluster](assets/create-rke-harvester-cluster.png)
+
+### Using Harvester RKE1 Node Driver in Air Gapped Environment
+
+RKE1 provisioning relies on the qemu-guest-agent to get the IP of the virtual machine, and docker to set up the RKE cluster.
+It may not be feasible to install qemu-guest-agent and docker in an air gapped environment.
```

Comment:
```suggestion
RKE1 provisioning relies on the `qemu-guest-agent` to get the IP of the virtual machine, and `docker` to set up the RKE cluster. However, it may not be feasible to install `qemu-guest-agent` and `docker` in an air gapped environment.
```